bless/aws_lambda/bless_lambda.py

```diff
@@ -124,7 +124,9 @@ def get_hostnames(service_name, service_instance, service_region, instance_id,
         hostname_prefixes.append('{cluster_name}-canary'.format(
             cluster_name=cluster_name))
     if onebox_name:
-        hostname_prefixes.append('{onebox_name}.onebox'.format(
+        hostname_prefixes.append('{onebox_name}-onebox.dev'.format(
+            onebox_name=onebox_name))
+        hostname_prefixes.append('*-{onebox_name}-onebox.dev'.format(
```

Comment:
> "valid principals" is a string containing zero or more principals as
> strings packed inside it. These principals list the names for which this
> certificate is valid; hostnames for SSH_CERT_TYPE_HOST certificates and
> usernames for SSH_CERT_TYPE_USER certificates. As a special case, a
> zero-length "valid principals" field means the certificate is valid for
> any principal of the specified type.

It looks like there's some confusion about if wildcards should be accepted. I think we'll need to test it out with.